Sources/SwiftSyntax/SyntaxNodes.swift.gyb.template

```diff
@@ -78,28 +78,7 @@ public struct ${node.name}: ${base_type}Protocol, SyntaxHashable {
 %           end
 %         end
     public init?<Node: SyntaxProtocol>(_ syntaxNode: Node) {
-%         token_choices = []
-%         other_choices = []
 %         for choice in child.node_choices:
-%           if choice.is_token():
-%             token_choices += [choice]
-%           else:
-%             other_choices += [choice]
-%           end
-%         end
-%
-%         if token_choices:
-      if let tok = syntaxNode.as(TokenSyntax.self) {
```

Comment:
We should be able to have `init(_: TokenSyntax)` initializer now.

Reply:
Good catch. Thanks